src/Core/Models/Table/Send.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Bit.Core.Enums;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.Models.Table
+{
+    public class Send : ITableObject<Guid>
+    {
+        public Guid Id { get; set; }
+        public Guid? UserId { get; set; }
+        public Guid? OrganizationId { get; set; }
+        public SendType Type { get; set; }
+        public string Data { get; set; }
+        public string Key { get; set; }
```

Comment:
Correct me if I'm wrong, but isn't the key only kept client-side, or is this a different key?